active/community.md

```diff
@@ -0,0 +1,75 @@
+# WG Charter Template
+
+## Scope of responsibilities
+
+This group exists to support the official community platforms and those in elevated roles on those platforms.
+
+The Forum and the Discord are core parts of the overall community and those that help and moderate these platforms take significant time from their day to help others. We can support them through training and taking action to make their lives easier.
+
+The broad goals of the group are:
+  - facilitate and centralise discussion between moderators and admins on the various platforms
+  - coordinate efforts for training and processes around moderation
+  - provide support to improve the platforms to lighten moderation and admin workloads
+  - provide training and support to the wider community on how to use the platforms effectively
+
+- What actions are you proposing the WG be allowed to take directly?
+  - Creation and management of processes related to community platforms.
+    - Adding/removing moderators & other roles
```

Comment:
I'm a bit wary of delegating this. I think what we have done with the Discord has worked well so far and I'm not sure people would be happy with having mods "imposed" on them. 

I'm not so sure about the forum. One issue is that I think only Andrew has full access, right?

Reply:
That's fair, I could have worded this better, but initial idea was to simply formalize what we already do with a tiny bit more regularity. ie having the WG prompt the mod teams to review any new moderators, ask if anyone wants to step down at regular periods (eg 6 months)

Definitely not going to be imposing anyone on any community